packages/calypso-e2e/src/lib/pages/jetpack-forms-inbox-page.ts

```diff
@@ -0,0 +1,84 @@
+import { Page } from 'playwright';
+import { envVariables } from '../..';
+
+/**
+ * Page representing the Jetpack Forms Inbox page. (The "Inbox", non-"Classic" view for forms responses.)
+ */
+export class JetpackFormsInboxPage {
```

Comment:
**suggestion:non-blocking** - this seems to be simply the "Feedback" page in the sidebar. I thought this page represented the page under SidebarComponent > Inbox which was not the case.

What if this POM was renamed to be simply "FeedbackPage"?